### DISCUSSION TITLE
Can we get to the bottom of the resolution issue?

### DISCUSSION THREAD

**Discussion by mdehrlich** [GerryDazoo/Slinger#552] [General]
My cable box sends a fixed 1080i signal to the slingbox.  I can get the full 1920X1080 with a resolution setting of 16.  The slingbox must be downsampling to get to lower resolutions.  I get 1280X540 with resolution settings of 15 or 12.  I get blah with any other setting.  Is there some other parameter that might tell the slingbox to give me 16:9 output?

**Comment by GerryDazoo** [comment]
Is "blah" a technical term? The short answer is it depends on the Slingbox. But on many of the resolutions the SB sends an anamorphic picture (blah, I'm assuming, does it all look scruched up?), In that case just set the output of your viewer to the correct aspect ratio and you should be golden.

**Reply by mdehrlich** [reply]
Blah is an unusable low quality image that I am not interested in watching.  Thanks for response.
